pkg/types/gcp/lbuid.go

```diff
@@ -0,0 +1,13 @@
+package gcp
+
+import (
+	"crypto/md5"
+	"fmt"
+)
+
+// GenerateLoadBalancerUID generates a UID to use for GCP service load balancers.
+// The UID is based on a hash of the infraID.
+func GenerateLoadBalancerUID(infraID string) string {
```

Comment:
same as https://github.com/openshift/installer/pull/2045/files#r305513919

currently this is confusing as `GenerateLoadBalancerUID` in `types` package seems odd. and this is actually an prefix for k8s gcp cloud provider.. we can keep it in the package but i think it should be renamed to something that is closer to `this is used by k8s cloud provider` ...

Reply:
I agree, the naming is not great... maybe it should be something like `CloudControllerUID`?

Reply:
yeah, that would be better.